pkgs/top-level/perl-packages.nix

```diff
@@ -11188,6 +11481,34 @@ let
     };
   };
 
+  LEOCHARRECLI = buildPerlPackage {
+    pname = "LEOCHARRE-CLI";
+    version = "1.19";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/L/LE/LEOCHARRE/LEOCHARRE-CLI-1.19.tar.gz";
+      sha256 = "37835f11ee35326241b4d30368ae1bc195a50414b3662db3e13b865bd52fcde9";
+    };
+    propagatedBuildInputs = [ FileWhich Filechmod LEOCHARREDEBUG Linuxusermod YAML ];
+    meta = {
+      description = "Useful subs for coding cli scripts";
+      license = with lib.licenses; [ artistic1 gpl1Plus ];
+    };
+  };
+
+  LEOCHARREDEBUG = LEOCHARREDebug;
```

Comment:
Isn't this a different CPAN package? https://metacpan.org/dist/LEOCHARRE-DEBUG

(While the above does say its deprecated and prefer to use https://metacpan.org/pod/LEOCHARRE::Debug instead, they're still distinct CPAN dists, which might lead to confusion later on esp when versions are considered.)

Reply:
Your right :) for some reason in my head the newer LEOCHARRE::Debug was backwards compatible with the older version but right now I cant find any reason to support my idea.

I'll remove this an for the pull request